### ISSUE TITLE
Litepaper Orderbook Section

### ISSUE THREAD

**Issue by Brando753** [Sifchain/sifnode#2867]
Missing implementation for order books In the Sifchain Lite Paper, it is mentioned that "Sifchain uses both an order book and a CLP for trade completion". However, the code base does not have any implementation related to order books. Either todo's should be added to support orderbooks, or the paper should be updated.

**Comment by pandaring2you** [comment]
@jkbrooks - do we want to remove this from the litepaper? or just put in a disclaimer that this is aspirational? cc @jon-sif

**Comment by pandaring2you** [comment]
from Jazear: ah then yea let's just deprecate the Litepaper for this audit
peggy.sifchain.finance is the only definitive document

Jordan: If they ask for high level documentation that the litepaper covers that peggy.sifchain.finance doesnt cover, I suppose we can add the CLP and IBC sections to docs.sifchain.finance

**Comment by pandaring2you** [comment]
Jazear: "you should definitely tell Certik that the definitive docs are https://peggy.sifchain.finance" . @sifmoon - can we deprecate this portion of the litepaper to satisfy certik?

**Comment by sifmoon** [comment]
Yeah sure. Is this something I need to assist with, or are you good to go in doing this?
